### DISCUSSION TITLE
SpeedyBee F405 mini wing and SmartAudio

### DISCUSSION THREAD

**Discussion by mazi14** [iNavFlight/inav#10359] [Support]
I have bought new FC SpeedyBee F405 mini wing. Flashed iNav 7.1.2. I have got working everything except SmartAudio function for VTx. I connected dedicated wire from VTx to TX5 and set it in Ports configuration page:
![VTX](https://github.com/user-attachments/assets/87021b57-aaab-450f-aacc-991630066780)

![SpeedyBee ports](https://github.com/user-attachments/assets/7bb42dc9-9553-4557-8958-1332a112593e)

I set band and channel to E4, but OSD shows A2, which confuse me. Goggles are receiving video on E4 channel, so only OSD information is wrong. I want to remotely control VTx power level only.
![VTx channel](https://github.com/user-attachments/assets/80168382-c085-4a1a-a18f-b215998799ce)

I used this VTx with FC Mater F411 wing (iNav 2.6.0) earlier and SmartAudio worked on SoftSerial port there just fine.

Am I doing something wrong?

**Comment by ebochat** [comment]
```
Hello, I have recently struggled with this aswell, make sure to do this : https://www.youtube.com/watch?v=vvetICkhmkk
I have managed to have correct channel/band readings, and can switch VTX power from a switch on the radio through programming entries in the programming tab.
Your VTX should have an additional SA output which can be plugged into a free UART on your FC. (My TBS Unify 32 DP has a SA output that I plugged into the TX2 output of my Speedybee f405 WING board).
```
^ Not deleting so that my mistake can help someone else. But I was confusing SmartAudio with SmartPort. Obviously they are not the same thing and in my case I did not have to worry about SmartPort or SoftSerial at all since using UART5 on the VTX plug and play input from the FC was enough to have vtx telemetry. I ended up rewiring my SA output into the FC's tx5 input, which fits right next to the other vtx wires (gnd,9v,vtx). We both have the same wiring in that regard.

Coming back to your issue. 

> OSD shows A2

This happened to me as I was desperately fiddling with those commands : 

set vtx_smartaudio_early_akk_workaround = [ON:OFF]
set vtx_smartaudio_alternate_softserial_method = [ON:OFF]
set vtx_softserial_shortstop = [ON:OFF]
set vtx_smartaudio_stopbits = [1:2]

They are detailed here : https://github.com/iNavFlight/inav/blob/master/docs/VTx.md and here https://github.com/iNavFlight/inav/blob/7.1.2/docs/Settings.md.

Have you tried your luck with those?

**Reply by Hamstarotto** [reply]
could you maybe share your settings for the vtx with me please? I am using the same hardware as you and its driving me nuts, as i cant seem to find the right combination of them. 
Thanks in advance

**Comment by mazi14** [comment]
Hello, I have a question regarding FC SpeedyBee F405 mini wing. There is iNav 7.1.2 flashed in. I log telemetry data in my FrSky Taranis Q X7 transmitter and among oher telemetry sensors there is one which has label "0460". Does anybody have an idea, what do these values mean? I have already asked FC producer for help, but their answer was, that this is a bug and they recommended me to try discover all sensors in transmitter again. After I did it, nothing has changed.
![20250124_105941](https://github.com/user-attachments/assets/545765b9-e1e4-47c4-abf2-5aeaf1d359ec)
